src/modules/billing/controllers/invoices.js

```diff
@@ -8,13 +8,19 @@ const invoiceIsPartOfSentBatch = invoice => {
   return billingBatch.status === BATCH_STATUS.sent;
 };
 
+const invoiceIsRebill = invoice =>
+  invoice.rebillingState !== null;
+
 const patchInvoice = async request => {
   try {
     const { invoiceId } = request.params;
     const invoice = await invoiceService.getInvoiceById(invoiceId);
     if (!invoiceIsPartOfSentBatch(invoice)) {
       return Boom.conflict('Cannot update invoice that is not part of a sent batch');
     }
+    if (invoiceIsRebill(invoice)) {
```

Comment:
Is this right? Wouldn't we want to patch the invoice as part of the CM refresh routine?

Reply:
This is preventing it from being flagged, rather than something that happens during the billing process.

The UI should not present invalid invoices for flagging, but this is an additional safety check.